index.js

```diff
@@ -21,6 +24,7 @@ var argv = yargs
 
     .alias('v', 'verbose')
     .describe('v', 'Be more chatty')
+    .default('v', false)
```

Comment:
I set the default value for `verbose` here. I had considered just propagating the `argv.verbose` all the way through but I figured the `const` was probably useful.